src/batch/src/executor/row_seq_scan.rs

```diff
@@ -34,18 +34,17 @@ use risingwave_storage::table::storage_table::{RowBasedStorageTable, StorageTabl
 use risingwave_storage::table::{Distribution, TableIter};
 use risingwave_storage::{dispatch_state_store, Keyspace, StateStore, StateStoreImpl};
 
-use crate::executor::monitor::BatchMetrics;
 use crate::executor::{
     BoxedDataChunkStream, BoxedExecutor, BoxedExecutorBuilder, Executor, ExecutorBuilder,
 };
 use crate::task::BatchTaskContext;
 
 /// Executor that scans data from row table
-pub struct RowSeqScanExecutor<S: StateStore> {
+pub struct RowSeqScanExecutor<S: StateStore, C: BatchTaskContext> {
     chunk_size: usize,
     schema: Schema,
     identity: String,
-    stats: Arc<BatchMetrics>,
+    context: Option<C>,
```

Comment:
Why we use an `Option` here?

Reply:
Because some executors used in the test such as (compute/test/integration_test) and in this test we don't have a valid context.

'Option' is used in some cases we can't give a valid context. The context is unnecessary to execute the executor (it is just used to collect metrics, 'None' means not to collect metrics).

Reply:
Hmm, I think the `Option` of `get_task_metrics` is good enough to indicate whether we need to collect metrics, and we don't need another `Option` here. For tests, I think the correct way is to create a `MockBatchTaskContext` rather passing an `Option`.